src/types.ts

```diff
@@ -8,6 +8,8 @@ import { Bid } from "./zAuction";
 export interface Config {
   /** The zNS Subgraph URL */
   subgraphUri: string;
+  /** Subgraph URIs for any DEX we check in the resolver process */
+  dexSubgraphUris: string[];
```

Comment:
Wouldn't this assume that all DEX's use the same subgraph schema? Can we really assume this?

Reply:
There will have to be changes for adding more DEX subgraphs but some have copied, or very mildly modified the code from Uniswap so it's a good place to start

Reply:
Okay, but do we really want to have it take an array of Uri's then? Wouldn't there need to be a different implementation for each Uri?

Reply:
Yes but I also imagined that the process of grabbing a specific DEX subgraph out of this array to be kept internal to this process so having it as an array in the config made more sense to me then a series of individual properties, but it's only for one protocol in this story anyways and has been changed to reflect that so this is more a discussion on designing the next steps that include checking several DEX protocols anyways